iac-extensions/docker/src/test/resources/hado-lint/docker-file.docker

```diff
@@ -0,0 +1,81 @@
+# MySQL Server with Apache and phpmyadmin
```

Comment:
I think that not all lines are needed here. Please try to use a minimum example as corresponding reports contain only a few issues related to this file